components/support/base/src/main/java/mozilla/components/support/base/ids/SharedIdsHelper.kt

```diff
@@ -39,3 +42,39 @@ object SharedIdsHelper {
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal fun clear(context: Context) = ids.clear(context)
 }
+
+/**
+ * Post a notification to be shown in the status bar.
+ *
+ * Uses a unique [String] tag instead of an [Int] id like [NotificationManager.notify].
+ */
+fun NotificationManager.notify(context: Context, tag: String, notification: Notification) {
```

Comment:
Could we also mention in all the docs that this will use the existing ID for the tag if it exists?

My minor worry is that someone will use this on two different notifications with the same ID, and without realizing that it'll overwrite the previous notification.

Maybe also reference the SharedIdsHelper? e.g. "See also: [SharedIdsHelper]."

Reply:
OK, I'll document this.  Thanks,